src/main/java/org/fogbowcloud/saps/engine/core/catalog/jdbc/exceptions/JDBCCatalogException.java

```diff
@@ -0,0 +1,10 @@
+package org.fogbowcloud.saps.engine.core.catalog.jdbc.exceptions;
+
+public class JDBCCatalogException extends Exception {
+
```

Comment:
why a new class for it?

Reply:
I added a fixme to remove this exception.